docker-image/v1.12/arm64/debian-azureblob/Gemfile.lock

```diff
@@ -152,4 +152,4 @@ DEPENDENCIES
   oj (= 3.11.0)
 
 BUNDLED WITH
-   2.1.4
+   2.2.3
```

Comment:
Shouldn't this be `2.2.6`?

Reply:
The latest ruby:alpine image does not include bundler 2.2.6. It just includes bundler 2.2.3.